tests/__snapshots__/metadata.spec.ts.snap

```diff
@@ -58,7 +58,7 @@ exports[`metadata > determineBrowserData > should return that the browser name a
 {
   "browser": {
     "name": "configCapabilities browser name",
-    "version": "configCapabilities browser version",
+    "version": "No metadata.browser.version available",
```

Comment:
Are you sure about this? I mean should this really be the expected value in the test?
If so, all fine, I am just curious and a bit confused 😅 .

Reply:
Right, there is ConfigCapabilities interface in use that includes browserName only, so the current logic is to determineBrowserData:
1) by current capabilities (interface includes both browserName & browserVersion)
2) if there is no "currentCapabilities" - by ConfigCapabilities (includes only browserName)
3) if  there is no 1 && 2 - use metadata info

So the test with this snapshot covers the second case

Reply:
Thanks @HannaTarasevich for the explanation. Now I got it and learned something, thanks.